shared-bindings/audiofilters/Filter.c

```diff
@@ -68,9 +69,9 @@
 //|               time.sleep(5)"""
 //|         ...
 static mp_obj_t audiofilters_filter_make_new(const mp_obj_type_t *type, size_t n_args, size_t n_kw, const mp_obj_t *all_args) {
-    enum { ARG_filter, ARG_mix, ARG_buffer_size, ARG_sample_rate, ARG_bits_per_sample, ARG_samples_signed, ARG_channel_count, };
+    enum { ARG_filters, ARG_mix, ARG_buffer_size, ARG_sample_rate, ARG_bits_per_sample, ARG_samples_signed, ARG_channel_count, };
     static const mp_arg_t allowed_args[] = {
-        { MP_QSTR_filter, MP_ARG_OBJ | MP_ARG_KW_ONLY,  {.u_obj = MP_OBJ_NULL} },
+        { MP_QSTR_filters, MP_ARG_OBJ | MP_ARG_KW_ONLY, {.u_obj = MP_OBJ_NULL} },
```

Comment:
`.u_obj = mp_const_empty_tuple`

Reply:
I plan on changing the default to `None` rather than an empty tuple. The only problem I potentially see here is if a user decides to iterate through `effect.filters` without testing it (ie: `if effect.filters: ...`. It will throw "TypeError: 'NoneType' object is not iterable" if that is the case, whereas an empty tuple wouldn't.

If you would prefer the empty tuple to avoid that situation, let me know.

Reply:
seems like if you're going for compatibility, then assigning `None` or a single `Biquad` would have to retain the current behavior when the property is fetched. On the other hand, you quickly identified that the ergonomics are not great for generic code when trying to be compatible. That's why I thought it was worth considering going ahead and breaking compatibility with this change.